packages/neuron-wallet/src/store/WalletStore.ts

```diff
@@ -0,0 +1,79 @@
+import BaseStore from './store'
+
+const walletDBName = 'WalletDB'
+
+const keyWalletName = 'WalletName'
+
+// wallet name is exist
+const WalletStoreError = {
+  ExistKey: 'There is wallet named ',
+}
+
+export interface Wallet {
+  name: string
+  keystore: keyStore
+}
+
+export interface keyStore {
+  // TODO
+}
+
+export default class WalletStore extends BaseStore {
+  constructor() {
+    super({
+      name: walletDBName,
+    })
+  }
+
+  private getWalletNameList(): string[] {
+    return this.get(keyWalletName, [])
+  }
+
+  saveWallet = (walletName: string, wallet: Wallet) => {
+    if (this.store.has(walletName)) {
+      throw new Error(WalletStoreError.ExistKey + walletName)
+    } else {
+      let nameList = this.getWalletNameList()
+      this.save(walletName, wallet)
+      nameList = nameList.concat(walletName)
+      this.save(keyWalletName, nameList)
+    }
+  }
+
+  getAllWallets(): Wallet[] {
+    const walletList: Wallet[] = []
+    const nameList = this.getWalletNameList()
+    nameList.forEach(name => walletList.push(this.getWallet(name)))
+    return walletList
+  }
+
+  getWallet(walletName: string): Wallet {
+    return this.get(walletName)
+  }
+
+  renameWallet(newName: string, oldName: string) {
+    const wallet = this.getWallet(oldName)
+    wallet.name = newName
+    this.saveWallet(newName, wallet)
```

Comment:
if wallet includes name, why use two params

Reply:
oldName is a index.It`s a file system nor db,so I should save a index list to find wallet.